From 4391d63651027968eb833e8c44740752bc6f0c98 Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Thu, 25 Jun 2020 09:03:16 +0200
Subject: [PATCH 16/40] include/prereq-build.mk: add ovsdb-tool dependency

OpenSync requires a host installation of ovsdb-tool. Add this to the
required commands.

If the tool is missing, users will be presented witht he following line.

-> Build dependency: Please install the 'ovsdb-tool' package

Signed-off-by: John Crispin <john@phrozen.org>
---
 include/prereq-build.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/prereq-build.mk b/include/prereq-build.mk
index 501739968c..ea22510bfd 100644
--- a/include/prereq-build.mk
+++ b/include/prereq-build.mk
@@ -140,6 +140,9 @@ $(eval $(call SetupHostCommand,git,Please install Git (git-core) >= 1.7.12.2, \
 $(eval $(call SetupHostCommand,file,Please install the 'file' package, \
 	file --version 2>&1 | grep file))
 
+$(eval $(call SetupHostCommand,ovsdb-tool,Please install the 'ovsdb-tool' package, \
+	ovsdb-tool -V 2>&1 | grep vSwitch))
+
 $(STAGING_DIR_HOST)/bin/mkhash: $(SCRIPT_DIR)/mkhash.c
 	mkdir -p $(dir $@)
 	$(CC) -O2 -I$(TOPDIR)/tools/include -o $@ $<
-- 
2.25.1

